pandas/core/indexes/datetimelike.py

```diff
@@ -478,6 +478,7 @@ def repeat(self, repeats, axis=None):
         nv.validate_repeat(tuple(), dict(axis=axis))
         freq = self.freq if is_period_dtype(self) else None
         return self._shallow_copy(self.asi8.repeat(repeats), freq=freq)
+        # TODO: dispatch to _eadata
```

Comment:
What's blocking this right now?

Reply:
I think nothing, just forgot to change before pushing.  Will do that now.

Reply:
Actually no, this needs to wait until DTI._data is a DTA